content/md/first-steps/manage-product-identifiers.md

```diff
@@ -0,0 +1,97 @@
+
+# Manage your products with the Identifier attribute
+
+## Description of the Identifier attribute
+
+The identifier is an attribute with a few specific characteristics:
+- Right now, only one identifier attribute is created by default. It is always called “SKU” as the attribute code can't be modified. Only its label can be modified.
+- The value of this attribute must be unique per product
+- This attribute is not localizable (per locale) nor scopable (per channel)
+
+::: warning
+Starting in September, the identifier attribute will evolve:
+- SKU will soon become optional, meaning this field can stay empty when creating a new product
+- This attribute can also be deleted from family attributes and be unselected from being required for product completeness.
+:::
+
+## Search for products with the identifier attribute
+
+One of the big benefits of the identifier attribute is to be able to identify your products by all means:
+On the PIM, you can search for a product with its identifier in two different ways: (a) use the search bar at the top of the product grid if you search for one specific product, (b) use the filter on the left with the in-list function if you want to display a list of several products.
+
+## Update your products with imports
+
+When exporting your products from the PIM, your downloaded files will contain an “ SKU “ column.
+Once you have modified your product details in the file, ensure the “SKU” column is still included in the document as it will help identify your products and update them.
+
+::: info
+Exported files now contain a “UUID” column. It is now the best way to identify your products as this technical product identifier is immutable, and will work even if you have products without, SKU. [Learn more in this section](manage-product-identifiers.html#manage-your-products-with-the-uuid)
+:::
+
+## Update your products with the API
+
+8 API endpoints are available to help you identify your products, update them, or even delete them. The identifier attribute is the key entry to identify specific products from your PIM.
+
+Please refer to [api.akeneo.com](https://api.akeneo.com/) for more details.
+
+::: warning
+8 new API endpoints have been created, working with “UUID” product identifiers, which is the most future-proof way to identify products, especially if you have or plan to have products without SKU.
+:::
+
+# Manage your products with the UUID
+
+## Description of the UUID, the new technical product identifier
+
+Introduced in September 2022, the UUID is a technical product identifier assigned to every product in the PIM.
+
+Here are some characteristics of this identifier:
+- The UUID is **not an attribute** but a technical identifier. Therefore, it won’t be displayed as a product value unlike the identifier attribute.
+- The UUID is **immutable**, meaning it can’t be modified or deleted after the product creation unlike the identifier attribute
+- The value of each UUID is unique in the PIM
```

Comment:
```suggestion
- The value of each UUID is unique in a PIM
```

Reply:
Another PIM instance could have the same UUID for another product even if the probability is low